lib/src/commands/packages.dart

```diff
@@ -11,7 +11,7 @@ import 'package:very_good_cli/src/cli/cli.dart';
 /// {@endtemplate}
 class PackagesCommand extends Command<int> {
   /// {@macro packages_command}
-  PackagesCommand({Logger? logger}) {
+  PackagesCommand({@visibleForTesting Logger? logger}) {
```

Comment:
Is this for test only?

Reply:
Oops, I've realized my mistake. Sorry, I'll be making the required changes now.